src/config/samples/policy.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     image: projects.registry.vmware.com/cnsi/inspector:0.3
     kubebenchImage: projects.registry.vmware.com/cnsi/kubebench:0.3
     riskImage: projects.registry.vmware.com/cnsi/risk:0.3
-    governorImage:
+    workloadScanner: projects.registry.vmware.com/cnsi/workloadscanner:0.3
```

Comment:
I don't see inspectionpolicy_types.go changes here, CRD bases have to be generated from types.

https://github.com/vmware-tanzu/cloud-native-security-inspector/blob/main/src/api/v1alpha1/inspectionpolicy_types.go